src/pytest_split/algorithms.py

```diff
@@ -0,0 +1,106 @@
+from typing import TYPE_CHECKING
+import heapq
+from collections import namedtuple
+
+if TYPE_CHECKING:
+    from typing import List, Dict, Tuple
+    from _pytest import nodes
+
+
+ALGORITHMS = ["duration_based_chunks", "least_duration"]
+
+test_group = namedtuple("test_group", "selected, deselected, duration")
```

Comment:
Now that we are using typehints, could you come up with a data structure which would have the type hints for the members? E.g. [`typing.NamedTuple` ](https://docs.python.org/3/library/typing.html#typing.NamedTuple)could do.

Reply:
Good suggestion, didn't know it existed.